javascript/canvas-zoom.js

```diff
@@ -398,6 +405,10 @@
     function toggleBrushPanel(openUnderMouse = false) {
       const colorID = getTabId();
 
+      if (colorID === inpaintID) {
```

Comment:
Clears errors in the console if you click on the color panel call in the inpaint mod